R/RagGrid.R

```diff
@@ -11,7 +11,7 @@
 #' @importFrom htmltools tags htmlDependency
 #'
 #' @export
-aggrid <- function(data, options=list(), licenseKey=NULL, width = NULL, height = NULL, elementId = NULL) {
+aggrid <- function(data, options=list(), licenseKey=NULL, colOpts=list(), width = NULL, height = NULL, elementId = NULL) {
```

Comment:
Can we change the order ? So i can call grid like this 

aggrid(data, gridOptions, columnOptions, licenseKey ...)

Reply:
OK